src/shims/env.rs

```diff
@@ -23,10 +23,14 @@ pub struct EnvVars<'tcx> {
 impl<'tcx> EnvVars<'tcx> {
     pub(crate) fn init<'mir>(
         ecx: &mut InterpCx<'mir, 'tcx, Evaluator<'tcx>>,
-        excluded_env_vars: Vec<String>,
+        mut excluded_env_vars: Vec<String>,
     ) -> InterpResult<'tcx> {
+        let target_os = ecx.tcx.sess.target.target.target_os.as_str();
+        if target_os == "windows" {
+            // Exclude `TERM` var to avoid terminfo trying to open the termcap file.
```

Comment:
Please also reference https://github.com/rust-lang/miri/issues/1013 here.

Reply:
I tried that, but the build fails due to a type mismatch error inside the match statement.